<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control</class>
 <widget class="QMainWindow" name="Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_temp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Temperature（℃）</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>480</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_temp_end">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>470</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Stop Recording</string>
    </property>
   </widget>
   <widget class="QSlider" name="sld_open">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>301</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>75</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_open">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Time of open（s）</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_open_show">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>410</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>0.75s</string>
    </property>
   </widget>
   <widget class="QSlider" name="sld_ratio">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>301</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_duty">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Control</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status_pic">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ratio_show">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>1.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Temperature Range：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_y_max">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>50.0℃</string>
    </property>
   </widget>
   <widget class="QSlider" name="sld_x_scale">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>541</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sld_y_center">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>20</width>
      <height>311</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>150</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="graphFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>631</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ratio_present">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>0.40</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Keep Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_x_present">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>1s/格</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_center">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Temperature Center：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_center_show">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>0.0℃</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_temp_pause">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>440</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Pause Recording</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ratio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Set Duty ratio（close/open）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_temp_start">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>410</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Start Recording</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ratio_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Duty ratio at present：</string>
    </property>
   </widget>
   <widget class="QSlider" name="sld_y_scale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>19</width>
      <height>311</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scale_show">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>100.0℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_y_min">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>-50.0℃</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>470</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Keep Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_y_mid">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>0.0℃</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
